cmd/hz/generator/package.go

```diff
@@ -142,7 +142,8 @@ func (pkgGen *HttpPackageGenerator) Generate(pkg *HttpPackage) error {
 
 	if pkgGen.CmdType == meta.CmdClient {
 		clientDir := pkgGen.IdlClientDir
-		clientDir = util.SubDir(clientDir, "hertz")
+		// support multiple service, so disable "clientDir/hertz"
+		// clientDir = util.SubDir(clientDir, "hertz")
```

Comment:
Why not just delete this line?

Reply:
Do it in another pr, this one has been merged~

Reply:
done~, in [pr](https://github.com/cloudwego/hertz/pull/511)